File: OpenApi2.0/push-openapi-php-demo/ListTags.php
<?php

include_once '../aliyun-php-sdk-core/Config.php';
include 'Push/Request/V20160801/ListTagsRequest.php';
use \Push\Request\V20160801 as Push;

$accessKeyId = "your accessKeyId";
$accessKeySecret = "your accessKeySecret";
$appKey = "your appKey";

$iClientProfile = DefaultProfile::getProfile("cn-hangzhou", $accessKeyId, $accessKeySecret);
$client = new DefaultAcsClient($iClientProfile);
$request = new Push\ListTagsRequest();

$request->setAppKey($appKey);

$response = $client->getAcsResponse($request);
print_r("\r\n");
print_r($response);

?>

File: OpenApi2.0/push-openapi-net-demo/ListTags.cs
using Aliyun.Acs.Core;
using Aliyun.Acs.Core.Exceptions;
using Aliyun.Acs.Core.Profile;
using Aliyun.Acs.Push.Model.V20160801;
using System;

namespace AlibabaCloud
{
    class ListTags
    {
        static void Main()
        {
            IClientProfile clientProfile = DefaultProfile.GetProfile("cn-hangzhou", "<your access key id>", "<your access key secret>");
            DefaultAcsClient client = new DefaultAcsClient(clientProfile);
            ListTagsRequest request = new ListTagsRequest();
            request.AppKey = <Your AppKey>;

            try
            {
                ListTagsResponse response = client.GetAcsResponse(request);
                Console.WriteLine("RequestId:" + response.RequestId);
                foreach (ListTagsResponse.TagInfo info in response.TagInfos)
                {
                    Console.WriteLine("TagName: " + info.TagName);
                }
                Console.ReadLine();
            }
            catch (ServerException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
            catch (ClientException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
                Console.ReadLine();
            }
        }

    }
}
